packages/st2mistral/debian/mistral-server.init

```diff
@@ -67,6 +68,19 @@ do_start()
   #   1 if daemon was already running
   #   2 if daemon could not be started
   lsb_running && return 1
+
+  # /run is a tmpfs mount, so we need to check the directory exists and create.
+  # Otherwise the service will not start following a reboot.
+  if [ ! -d "$RUN_DIR" ] ; then
+      if ! mkdir -p "$RUN_DIR" ; then
+          log_failure_msg "can't create $RUN_DIR"
+          exit 1
+      fi
+      if ! chown mistral:mistral "$RUN_DIR" ; then
+          log_warning_msg "can't chown $RUN_DIR"
+      fi
+  fi
```

Comment:
@jjm. @armab obviously suggests more elegant way, so it would be nice if it looked as above ^^ .
@armab `install` binary is always available and safe to use, since it's a part of **coreutils**.